### PR TITLE
Add documentation for "class" system field

### LINKED ISSUE #6915: [DOCS] TiddlerFields class Description field is missing
### Describe the bug

To reproduce 

https://tiddlywiki.com/#TiddlerFields ... `class` Description in second table is missing

### PR DESCRIPTION
**Pull request by hoelzro**
Fixes #6915

### CODE REVIEW

File: core/language/en-GB/Fields.multids [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: $:/language/Docs/Fields/
 _canonical_uri: The full URI of an external image tiddler
 bag: The name of the bag from which a tiddler came
 caption: The text to be displayed on a tab or button
+class: The CSS class applied to a tiddler when rendering it - see [[Custom styles by user-class]].  Also used for [[Modals]]
```
Comment (Jermolene) [inline core/language/en-GB/Fields.multids:6]: This file is part of the core plugin, and so will have to be updated via the master branch, and wait for the next version before release.

**Review by Jermolene — COMMENTED** [review]
Thanks @hoelzro

File: core/language/en-GB/Fields.multids (hunk 1 above)
Comment (hoelzro) [reply core/language/en-GB/Fields.multids:6]: Ok, thanks for the feedback @Jermolene!  What's the right way to proceed here?  Do I need to commit on what's currently on master and submit an PR against the master branch, then?  Or will my change just make it over into master the next time you merge `tiddlywiki-com` into master?

Comment (saqimtiaz) [reply core/language/en-GB/Fields.multids:6]: > Do I need to commit on what's currently on master and submit an PR against the master branch, then?

Correct.

Comment (hoelzro) [reply core/language/en-GB/Fields.multids:6]: Ok, done! I opened #6990 against `master`

Comment (hoelzro) [reply core/language/en-GB/Fields.multids:6]: BTW, if I have more questions about the workflow for contributing to the docs (eg. which tiddlers dictate the source/target branch), would you prefer I ask in https://github.com/Jermolene/TiddlyWiki5/discussions, talk.tiddlywiki.org, or somewhere else?

Comment (saqimtiaz) [reply core/language/en-GB/Fields.multids:6]: Github discussions would be good I think. Generally anything that is a part of the core itself (so the boot and core directory in the root of the repo and their contents) need to be changed as part of a TW release and therefore PRs need to be against master. The same applies for purely documentation updates that have to do with new features that are not yet part of a released TW version.

Documentation PRs that target existing features and tiddlers in the editions directory can go live immediately and should target the tiddlywiki-com branch.

**Comment by pmario** [comment]
@Jermolene -- We missed this one for v5.3.2 :/ 

@hoelzro -- As Jeremy wrote, it needs to be targetted to the master-branch

**Comment by hoelzro** [comment]
@pmario I opened #6990 against `master` - should I close this PR, then?

**Pull request closed without merging**